README.md

```diff
@@ -43,17 +55,11 @@ argNorm can also be installed through conda:
 conda install bioconda::argnorm
 ```
 
-## Supported tools
-
-Note that CARD RGI already uses ARO, thus there is no need to use argNorm.
+## Tutorial Video
 
-- [DeepARG](https://bench.cs.vt.edu/deeparg) (v1.0.2)
-- [ARGs-OAP](https://galaxyproject.org/use/args-oap/) (v3)
-- [ABRicate](https://github.com/tseemann/abricate) (v1.0.1) with NCBI (v3.6), ResFinder (v4.1.11), MEGARes (v2.0), ARG-ANNOT (v5), ResFinderFG (v2)
-- [ResFinder](https://bitbucket.org/genomicepidemiology/resfinder/src/master/) (v4.0)
-- [AMRFinderPlus](https://github.com/ncbi/amr) (v3.10.30)
+[![argNorm Tutorial](https://markdown-videos-api.jorgenkh.no/url?url=https%3A%2F%2Fyoutu.be%2Fvx8MCQ7gDLs)](https://youtu.be/vx8MCQ7gDLs)
 
-## Basic usage
+## Basic Usage
```

Comment:
I don't have a strong opinion on title case vs. normal case (title case does come across as a bit old-fashioned, but that's okay)

However, now there is a mix of title case vs. normal